composer.json

```diff
@@ -413,18 +386,7 @@
       "blt source:build:simplesamlphp-config"
     ],
     "post-drupal-scaffold-cmd": [
-      "cd docroot && patch -p1 <../patches/htaccess-saml.patch",
-      "patch -p1 < patches/htaccess-https.patch"
-    ],
-    "copy-saml": [
```

Comment:
Where has this functionality moved to? Future us thanks you.

Reply:
- The htaccess patches have been rolled into a single patch instead of multiple.
- `copy-saml` is provided by `blt source:build:simplesamlphp-config`
- No longer using any of the `release-it` stuff.